src/Magnum/Audio/Buffer.h

```diff
@@ -83,7 +89,27 @@ class Buffer {
              *
              * @requires_al_extension Extension @al_extension{EXT,double}
              */
-            StereoDouble = AL_FORMAT_STEREO_DOUBLE_EXT
+            StereoDouble = AL_FORMAT_STEREO_DOUBLE_EXT,
+
+            Quad8 = AL_FORMAT_QUAD8,
+            Quad16 = AL_FORMAT_QUAD16,
+            Quad32 = AL_FORMAT_QUAD32,
+
+            Rear8 = AL_FORMAT_REAR8,
+            Rear16 = AL_FORMAT_REAR16,
+            Rear32 = AL_FORMAT_REAR32,
+
+            Five_1Chnl8 = AL_FORMAT_51CHN8,
```

Comment:
@Squareys @mosra Yeah, I had no clue what to go with here. Both of those feel awkward too, but I'll go with whatever. I just want surround sound, dang it!